roles/matrix-riot-web/defaults/main.yml

```diff
@@ -32,6 +32,25 @@ matrix_riot_web_welcome_user_id: "@riot-bot:matrix.org"
 # Branding of riot web
 matrix_riot_web_brand: "Riot"
 
+# URL to Logo on welcome page
+matrix_riot_web_welcome_logo: "welcome/images/logo.svg"
+
+# URL of link on welcome image
+matrix_riot_web_welcome_logo_link: "https://riot.im"
+
+matrix_riot_web_welcome_headline: "_t('Welcome to Riot.im')"
+matrix_riot_web_welcome_text: "_t('Decentralised, encrypted chat &amp; collaboration powered by [matrix]')"
+
+# Links, shown in footer of welcome page:
+# [{"text": "Link text", "url": "https://link.target"}, {"text": "Other link"}]
+matrix_riot_web_welcome_footerlinks: false
```

Comment:
It's more consistent with the rest of the playbook to name this `matrix_riot_web_branding_authFooterLinks`. Similarly for the other variables which end up in the `config.json.j2` template.

If some variable serves multiple purposes or is used elsewhere (`welcome.html.j2`), it can probably be named differently.

Is `false` a good default value for this, indicating "do nothing special"? What if we send an empty list (`[]`)?
I assume you've tested this, by it doesn't hurt asking just to be sure :)